docs/src/dev-test-load.md

```
@@ -0,0 +1,15 @@
+# Load testing
+
+### 1. Update k6 tests
+
+   Update the `./test/load.js` file, to add your url for testing, and update other configurations in the same file.
+
+### 2. Run load.js
+
+   Run the following command to run the test.
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)